src/centralized_telescope/wrapper.rs

```diff
@@ -0,0 +1,38 @@
+use super::algorithm;
+use super::init::make_setup;
+use super::params::Params;
+use super::proof::Proof;
+use super::setup::Setup;
+use crate::utils::types::Element;
+
+/// The main ALBA struct with prove and verify functions.
+#[derive(Debug, Clone, Copy)]
+pub struct Wrapper {
+    setup: Setup,
+}
+
+impl Wrapper {
```

Comment:
Why not calling this CentralizedTelescope directly?

Reply:
I was thinking along the lines of: inside the `centralized_telescope` directory, `CentralizedTelescope` doesn't say much, so I called it `Wrapper` and put it in `wrapper.rs`. For users outside of `centralized_telescope`, however, it is re-exported as  `CentralizedTelescope`. What do you think?

Reply:
Sure